src/main/java/org/identifiers/cloud/ws/resourcerecommender/models/ScoreProviderEbi.java

```diff
@@ -0,0 +1,39 @@
+package org.identifiers.cloud.ws.resourcerecommender.models;
+
+import lombok.extern.slf4j.Slf4j;
+import org.identifiers.cloud.ws.resourcerecommender.api.data.models.ResolvedResource;
+import org.springframework.util.PatternMatchUtils;
+
+import java.util.regex.Pattern;
+
+/**
+ * Project: resource-recommender
+ * Package: org.identifiers.cloud.ws.resourcerecommender.models
+ * Timestamp: 2018-06-20 9:29
+ *
+ * @author Manuel Bernal Llinares <mbdebian@gmail.com>
+ * ---
+ *
+ * This Score Provider calculates its score taking into account whether a resource is hosted by EBI or not.
+ */
+@Slf4j
+public class ScoreProviderEbi implements ScoreProvider {
+    private boolean isEbi(final ResolvedResource resolvedResource) {
+        return PatternMatchUtils.simpleMatch("http*://*ebi.ac.uk/*", resolvedResource.getCompactIdentifierResolvedUrl());
```

Comment:
I'm curious what Spring's pattern matcher is doing behind the scenes. If it's converting to regular expressions, I think I would prefer this written as a regex from the start.